src/web/releases/ivy/8.0.1/unsafe.version

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
or maybe we introduce a new file for this?
- security-issue.json
- security.json
- ... ?

Reply:
would you prefer a complete new file (replacement for the unsafe.version) or an additional one?

Reply:
best would be to replace the existing. convert it directly on the server.
but maybe it is not worth.
its ok to me to merge it now.